metadata-service/factories/src/main/resources/application.yml

```diff
@@ -179,3 +179,6 @@ spring:
 springdoc:
   cache:
     disabled: true
+
+tests:
```

Comment:
is this clear enough? should we say datahubTests?

Reply:
metadataTests?

Reply:
Yeah. just anything more than simple tests which can also refer to unit tests